utils/vectordb/vector_db.py

```diff
@@ -25,6 +25,7 @@
 from langchain.text_splitter import CharacterTextSplitter, RecursiveCharacterTextSplitter
 from langchain_community.document_loaders import DirectoryLoader, UnstructuredURLLoader
 from langchain_community.vectorstores import FAISS, Chroma, Qdrant
+from langchain_milvus import Milvus
```

Comment:
@snova-jorgep , we don't need Milvus in each kit and this might cause error as most kits don't have Milvus as part of their requirements when used with a local venv. Should we add a condition here?

Reply:
@snova-petrojm - Let's not add a condition please. To fully integrate this, those kits (that use vector db class) requirements will need to change ie add milvus.

Reply:
updated local requirements